src/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.cs

```diff
@@ -81,7 +81,7 @@ public partial class DataTypeAttribute : System.ComponentModel.DataAnnotations.V
         public virtual string GetDataTypeName() { throw null; }
         public override bool IsValid(object value) { throw null; }
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(2496), AllowMultiple = false)]
+    [System.AttributeUsageAttribute((System.AttributeTargets)(2500), AllowMultiple = false)]
```

Comment:
The integer is here because the tool that generates the draft ref files has a bug. It is better if possible to write it out so humans can read it like the other file.

Reply:
Sure! Let me do it!

Reply:
Done!

Reply:
Sorry! removing my commit. Commited the wrong version.